src/integration-test/utils.ts

```diff
@@ -104,3 +105,70 @@ export function logTitle(text: string) {
 export function prettyUrl(url: string) {
   return chalk.blue.underline(url);
 }
+
+export async function getResponseJson(response: OriginalResponse, requiredStatus = 200) {
+  const responseBody = await response.text();
+  let json: any;
+  try {
+    json = JSON.parse(responseBody);
+  } catch (error) {
+    // No need to handle here
+  }
+  if (response.status !== requiredStatus) {
+    const msgParts = [
+      `Got ${response.status} instead of ${requiredStatus}`,
+      response.url,
+      responseBody,
+    ];
+    throw new Error(msgParts.join(`\n\n`));
+  }
+  if (!json) {
+    const msgParts = [
+      `Unable to parse json`,
+      `${response.url} => ${response.status}`,
+      responseBody,
+    ];
+    throw new Error(msgParts.join(`\n\n`));
+  }
+  return json;
+}
+
+export async function getAccessToken(config: Auth0) {
+  const { domain, audience, clientId, clientSecret } = config;
+  const body = {
+    audience,
+    client_id: clientId,
+    client_secret: clientSecret,
+    grant_type: 'client_credentials',
+  };
+  const url = `${domain}/oauth/token`;
+  const response = await originalFetch(url, {
+    method: 'POST',
+    headers: {
+      'content-type': 'application/json',
+    },
+    body: JSON.stringify(body),
+  });
+  const json = await getResponseJson(response);
+  return json.access_token as string;
+}
+
+export function getConfiguration(env?: ENV, silent = false): Config {
+  // Load bindings that represent configuration
+  const _env: ENV = env || process.env.NODE_ENV || 'development';
+  let config: any;
+  switch (_env) {
+    case 'staging':
+      config = require('./configuration.staging').default;
+      break;
+    case 'development':
+      config = require('./configuration.development').default;
+      break;
+    default:
```

Comment:
Hmm. Actually I feel like it should be added here. You can then deny it by not providing a config for production.